batch/batch/front_end/front_end.py

```diff
@@ -1065,8 +1065,12 @@ async def _create_jobs(userdata: dict, job_specs: dict, batch_id: int, update_id
             spec['env'].append({'name': 'AZURE_APPLICATION_CREDENTIALS', 'value': '/gsa-key/key.json'})
 
         cloudfuse = spec.get('gcsfuse') or spec.get('cloudfuse')
-        if not is_developer and user not in ('ci', 'test', 'test-dev') and cloudfuse is not None and len(cloudfuse) > 0:
-            raise web.HTTPBadRequest(reason='cloudfuse requests are temporarily not supported.')
+        if cloudfuse:
+            for config in cloudfuse:
+                if not config['read_only']:
+                    raise web.HTTPBadRequest(reason='Only read-only cloudfuse requests are supported.')
+                if config['mount_path'] == '/io':
+                    raise web.HTTPBadRequest(reason='Cloudfuse requests with mount_path=/io are not supported.')
```

Comment:
likewise include bucket name to aid debugging. Maybe just stick the whole `config` in there as a JSON string?